pulp_ansible/app/urls.py

```diff
@@ -35,7 +35,14 @@
 
 v3_urls = [
     path(
-        "collections/", views_v3.CollectionViewSet.as_view({"get": "list"}), name="collections-list"
+        "collections/",
+        views_v3.CollectionViewSet.as_view({"get": "list"}),
+        name="collections-list",
+    ),
+    path(
+        "artifacts/collections/",
+        views_v3.CollectionViewSet.as_view({"post": "create"}),
+        name="artifact-collection-create",
```

Comment:
@cutwater is there another name you imagined here?

Reply:
`collection-artifact-create` or `collection-artifact-upload` sounds better IMO.

Reply:
I went with `collection-artifact-upload`. Thank you.